uberfire-project/uberfire-project-backend/src/main/java/org/guvnor/common/services/project/backend/server/POMServiceImpl.java

```diff
@@ -97,6 +111,25 @@ private MavenRepository getRepository(final String baseURL) {
         return mavenRepository;
     }
 
+    private boolean write(Model model, org.uberfire.java.nio.file.Path pathToPOMXML, IOService ioService) {
+        ByteArrayOutputStream baos = new ByteArrayOutputStream();
+        try {
+            writer.write(baos, model);
+            ioService.write(pathToPOMXML, new String(baos.toByteArray(), StandardCharsets.UTF_8));
+            return true;
+        } catch (Exception e) {
+            return false;
+        } finally {
+            try {
```

Comment:
This bit of stream closing boilerplate can be removed if you use [try-with-resources](https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html), which closes the stream for you:

```
        try (ByteArrayOutputStream baos = new ByteArrayOutputStream()) {
            writer.write(baos, model);
            ioService.write(pathToPOMXML, new String(baos.toByteArray(), StandardCharsets.UTF_8));
        } catch (Exception e) {

        }
```

Reply:
Thanks for the tips :)